docs/userguide.md

```diff
@@ -275,12 +275,16 @@ df.write
 // as tested, setting to `150` is a good practice
 .option(JDBCOptions.JDBC_BATCH_INSERT_SIZE, 150)
 .option("dbtable", s"cust_test_select") // database name and table name here
-.option("isolationLevel", "NONE") // recommended to set isolationLevel to NONE if you have a large DF to load.
+.option("isolationLevel", "NONE") // set isolationLevel to NONE
 .option("user", "root") // TiDB user here
 .save()
 ```
 
-It is recommended that you set `isolationLevel` to `NONE` to avoid large single transactions which might lead to TiDB OOM.
+Please set `isolationLevel` to `NONE` to avoid large single transactions which might lead to TiDB OOM and also avoid the following error:
+
+```
+java.sql.SQLException: variable 'tx_isolation' does not yet support value: READ-UNCOMMITTED
```

Comment:
I'd rather not to pin specific error message on other system since they might change without notifying us. Just explain that TiDB has different isolation level design and specify one will cause sanity check error.

Reply:
agree